src/main/java/frc/robot/commands/DriveCommands.java

```diff
@@ -46,7 +46,13 @@ public static enum AlignToReefSelector {
 
           return new DriveToPose(s.drivetrain, pose, Swerve.getMaxSpeed() * 0.3);
         }),
-    ;
+
+    USING_LATERAL_ALIGNMENT(
+        (s, p) -> {
+          return Commands.sequence(
+              Commands.idle(s.drivetrain).until(() -> s.frontCamera.get().hasTargets()),
```

Comment:
There's now a [`VisionCommands.waitForAprilTag()`](https://github.com/NRG948/NRGRobot2025/blob/58ec8156d803b5a36f07ad5a662db6a93609b10a/src/main/java/frc/robot/commands/VisionCommands.java#L30) function to handle waiting for targets while handling the optional camera subsystem.
```suggestion
              VisionCommands.waitForAprilTag(s.aprilTag, s.drivetrain),
```